src/iris/webhooks/alertmanager.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import
+
+import datetime
+import logging
+import ujson
+from falcon import HTTP_201, HTTPBadRequest, HTTPNotFound
+
+from iris import db
+
+logger = logging.getLogger(__name__)
+
+
+class alertmanager(object):
+    allow_read_no_auth = False
+
+    def on_get(self, req, resp):
```

Comment:
If you leave on_get undefined, falcon will reply back with a HTTP 405 which is probably good enough